Plugins/DD4hep/include/Acts/Plugins/DD4hep/DD4hepConversionHelpers.hpp

```diff
@@ -0,0 +1,93 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2017-2018 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <csignal>
+
```

Comment:
Does this need an `#include <exception>`

Reply:
No this actually needs to go away again. Good catch.

Reply:
Should be gone.